hardinfo/util.c

```diff
@@ -1432,6 +1432,7 @@ gboolean hardinfo_spawn_command_line_sync(const gchar *command_line,
                                           GError **error)
 {
     shell_status_pulse();
+    DEBUG("Trying: %s", command_line);
```

Comment:
Mind if I leave it here? It slightly helps to discover how hardinfo works internally.